### DISCUSSION TITLE
使用 LaTeX 完成毕业论文——SDUFE thesis 开发记录

### DISCUSSION THREAD

**Discussion by giscus** [mslxl/mslxl.github.io#28] [Show and tell]
# tex/sdufe-thesis-in-latex/

最近因为课程作业的原因，不得以提前准备毕业论文的 LaTeX 模板。作为一个仅使用过 LaTeX 而未制作过模板的人来说，这其实挑战还挺大的。不如从 WHU-Thesis1（以下简称为whu） 开始魔改一份模板出来。 那么，WHU-Thesis 做了什么? 创建 Logger 首先 whu 为 \msg_new 等函数创建了一个别名 1234567%tag 简化-msg-模块的函数\cs_new:

https://blog.mslxl.com/tex/sdufe-thesis-in-latex/

**Comment by mslxl** [comment]
快开发完了才发现今年学校强制要求提交docx了。有些搞笑了。
